packages/cli/src/commands/pull.ts

```diff
@@ -69,19 +68,14 @@ function processArgs(client: Client) {
 }
 
 function parseArgs(client: Client) {
-  try {
-    const argv = processArgs(client);
+  const argv = processArgs(client);
 
-    if (argv['--help']) {
-      help();
-      return 2;
-    }
-
-    return argv;
-  } catch (err) {
-    handleError(err);
-    return 1;
+  if (argv['--help']) {
+    help();
+    return 2;
   }
+
+  return argv;
```

Comment:
This try-catch was changing how errors were handled. We should let errors bubble up to the CLI entry point where they are handled consistently.